pkg/infrastructure/attribute_manager.go

```diff
@@ -46,12 +51,27 @@ func readInfrastructure() (infraattributes.Infrastructure, error) {
 	if err != nil {
 		return attributes, err
 	}
+	if err := validateStructure(content, ValidationPath); err != nil {
```

Comment:
Shouldn't it be `err =`  instead of `err :=` because err is already defined?

Reply:
`err` here is scoped to the if statement